services/leaf_space_integration/contact_reservations/schemas.go

```diff
@@ -0,0 +1,54 @@
+package contact_reservations
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+var contactReservationSchema = map[string]*schema.Schema{
+	"id": {
+		Type:     schema.TypeString,
+		Computed: true,
+	},
+	"contact_state_id": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"leafspace_status": {
+		Type:     schema.TypeString,
+		Required: true,
+	},
+	"status": {
+		Type:     schema.TypeString,
+		Computed: true,
```

Comment:
So I just tested and it looks weird: it seems `status=leafspace_status` and leafspace_status does not exist after the creation of the object. Can you check with the team because otherwise right now it doesn't work as intended

Reply:
we just have a status https://docs.leanspace.io/docs/api/branches/dev/leafspace/operations/create-a-connection

Reply:
https://leanspace-io.slack.com/archives/C057A5XBMQA/p1692631698826889